example/requirements.txt

```diff
@@ -0,0 +1,2 @@
+Django<2
```

Comment:
The title of this PR is `Example project: Require django>2`, but here I see `Django<2` :thinking:

Reply:
typo 😭